src/assets/scripts/client/aircraft/AircraftInstanceModel.js

```diff
@@ -2272,49 +1259,146 @@ export default class Aircraft {
             return;
         }
 
+        this.updateAircraftTurnPhysics();
+
+        this.updateAltitudePhysics();
+
+        if (this.isOnGround()) {
+            this.trend = 0;
+        }
+
+        // SPEED
+        this.updateSpeedPhysics();
+
+        if (!this.position) {
+            return;
+        }
+
+        // Trailling
+        if (this.position_history.length === 0) {
+            this.position_history.push([
+                this.position[0],
+                this.position[1],
+                window.gameController.game_time() / window.gameController.game_speedup()
+            ]);
+            // TODO: this can be abstracted
+        } else if (abs((window.gameController.game_time() / window.gameController.game_speedup()) - this.position_history[this.position_history.length - 1][2]) > 4 / window.gameController.game_speedup()) {
+            this.position_history.push([this.position[0], this.position[1], window.gameController.game_time() / window.gameController.game_speedup()]);
+        }
+
+        // FIXME: is this ratio correct? is it 0.000514444 or 0.514444?
+        const scaleSpeed = this.speed * 0.000514444 * window.gameController.game_delta(); // knots to m/s
```

Comment:
Well, it depends which one was in use previously. I'm not sure what is expected from the subsequent use of this variable. As it is shown here, `scaleSpeed` will be in meters. If it is expected in kilometers, then `0.00051444444` is the correct value.

```
const SPEED = {
    KNOTS_TO_METERS_PER_SECOND: 1 / 1.94384
};
```
^this might also be a useful thing to add to make this easier, more clear, and more reusable.

Since we're talking about 3 orders of magnitude, it should be *extremely* obvious if it is wrong, as the aircraft would move at a speed of 3 *kilometers* per second as opposed to 3 *meters* per second.